frontend/scripts/react-components/tool/basemaps/basemaps.scss

```diff
@@ -25,10 +25,16 @@
     box-shadow: 0 1px 3px 0 rgba($black, 0.1), 0 1px 2px 0 rgba($black, 0.2);
     pointer-events: all;
 
+    &:hover, &:active {
+      background-color: $light-gray;
+    }
+
     > .icon {
       fill: $charcoal-grey;
       height: 16px;
       width: 16px;
+      margin: 0;
+      transform: translateY(1px);
```

Comment:
why using translate and not a margin-top?

Reply:
Good question. I think using a transform is cleaner. It’s only applied on the item. A margin shifts things around. Also it’s the true purpose of the property.